config/initializers/assets.rb

```diff
@@ -9,3 +9,9 @@
 # Precompile additional assets.
 # application.js, application.css, and all non-JS/CSS in app/assets folder are already added.
 # Rails.application.config.assets.precompile += %w( search.js )
+Rails.application.config.assets.precompile += %w( edit_new.css )
```

Comment:
@miranmirza usually all the assets in app/assets/stylesheets should be automatically loaded. Was it being fussy and not showing the styles?